### DISCUSSION TITLE
Openai Command

### DISCUSSION THREAD

**Discussion by grimr502** [XaviaTeam/XaviaBot#8] [General]
I love your work, can you please add a command with an openai api key that when I ask xavia anything it'll respond just like chatgpt?

**Comment by RFS-ADRENO** [comment]
tbh, I have already made that command, it just that I won't release it for now due to some limitation.

**Comment by grimr502** [comment]
Oh, that's sad. I just badly need that command. :<

**Reply by RFS-ADRENO** [reply]
If you want it that bad, dm me via [Facebook](https://www.facebook.com/Dungto213/)

**Reply by grimr502** [reply]
Notice my dm.
